PixiEditor/Models/ImageManipulation/BitmapUtils.cs

```diff
@@ -56,30 +57,14 @@ public static WriteableBitmap CombineLayers(int width, int height, params Layer[
                         throw new InvalidOperationException("Layers must not extend beyond canvas borders");
                     }
 
-                    for (int y = 0; y < layers[i].Height; y++)
```

Comment:
Layer width and height is smaller than document one, and thus it executes faster. Is there any reason you changed this?